client/asset/dcr/config.go

```diff
@@ -78,6 +78,7 @@ func loadConfig(settings map[string]string, network dex.Network) (*Config, *chai
 	if cfg.RPCCert == "" {
 		cfg.RPCCert = defaultRPCCert
 	}
+	cfg.RPCCert = dex.CleanAndExpandPath(cfg.RPCCert)
```

Comment:
Could be an else here, because you wouldn't need to try and clean the default path I don't think.

Reply:
Yes, wanting to keep things minimal.